package.json

```diff
@@ -43,5 +43,8 @@
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-0": "^6.16.0"
+  },
+  "dependencies": {
+    "prop-types": "^15.5.8"
```

Comment:
I would also bump the version of react under peerDependencies to ^0.14.9 || ^0.15.3, since those are the compatible version of the new standalone 'prop-types'. If you don't, and a user has an older version of react, they are going to get this warning: `Warning: Failed propType: Calling PropTypes validators directly is not supported by the 'prop-types' package. Use 'PropTypes.checkPropTypes()' to call them.`.